code/game/objects/items/eightball.dm

```diff
@@ -39,7 +39,7 @@
 /obj/item/toy/eightball/Initialize(mapload)
 	..()
 	if(prob(1))
```

Comment:
While you're here, you might want to also make this line check to see that it's not already haunted.

Remember the RNG subsystem that's being implemented, and how goof wanted to try a round where every call to prob() returns true?  Well that would cause an infinite loop here.